backend/src/utils/resend.rs

```diff
@@ -0,0 +1,69 @@
+use crate::routes::auth::register::EMAIL_REGEX;
+use crate::utils::error::Error;
+use actix_web::HttpResponse;
+use regex::Regex;
+use resend_rs::types::CreateEmailBaseOptions;
+use resend_rs::{Resend, Result};
+use std::option::Option;
+use std::path::Path;
+
+pub enum EmailTemplate {
+    HelloWorld,
+}
+
+impl EmailTemplate {
+    pub fn changeable_elements(&self) -> Option<Vec<String>> {
+        match self {
+            EmailTemplate::HelloWorld => None,
+        }
+    }
+    pub fn get_template_path(&self) -> String {
+        match self {
+            EmailTemplate::HelloWorld => "src/utils/resend_assets/hello_world.html".to_string(),
+        }
+    }
+}
+
+pub async fn send_email(
+    sender: String,
+    receivers: Vec<String>,
+    subject: String,
+    template: EmailTemplate,
+    changeable_elements: Option<Vec<(String, String)>>,
+) -> Result<HttpResponse, Error> {
+    let resend = Resend::default();
```

Comment:
Use `Resend::new("re_123456789");` and pass `Config.resend_api_key` so all envs are tracked there